en/controllers/middleware.rst

```diff
@@ -45,11 +70,11 @@ hook method will be called early in the request process, you can use the
 
     class Application extends BaseApplication
     {
-        public function middleware($middlewareStack)
+        public function middleware($middleware)
```

Comment:
It was already changed: https://github.com/cakephp/app/pull/507 so why changing it again?